_includes/payment-order-reversal.md

```diff
@@ -1,6 +1,27 @@
+{% capture techref_url %}{% include documentation-section-url.md href='/features/technical-reference' %}{% endcapture %}
+{% assign transactions_url = '/transactions' | prepend: techref_url %}
+{% assign operations_url = '/operations' | prepend: techref_url %}
+{% capture documentation_section %}{%- include documentation-section.md -%}{% endcapture %}
+
+{% if documentation_section contains "checkout/v3" %}
+
+## Reversal
+
+This transaction is used when a captured payment needs to be reversed.
+
+### Create reversal transaction
+
+If we want to reverse a previously captured amount, we need to perform
+`reversal` against the accompanying `href` returned in the
+`operations` list.
+
+{% else %}
+
 If we want to reverse a previously captured amount, we need to perform
 `create-paymentorder-reversal` against the accompanying `href` returned in the
-`operations` list. See the abbreviated request and response below:
+`operations` list.
+
+{% endif %}
```

Comment:
... and this under the `{% if documentation_section contains "checkout-v3" %}` ?